examples/diagnostics_and_criticism/Diagnosing_biased_Inference_with_Divergences.ipynb

```diff
@@ -2355,6 +2361,21 @@
     "plt.legend();"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Adapted from Michael Betancourt's post [Diagnosing Biased Inference with Divergences](https://mc-stan.org/users/documentation/case-studies/divergences_and_bias.html), January 2017"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* Updated by Agustina Arroyuelo in February, 2018 ([pymc#2861](https://github.com/pymc-devs/pymc/pull/2861))\n",
+    "* Updated by [@CloudChaoszero](https://github.com/CloudChaoszero) in January, 2021 ([pymc-examples#25](https://github.com/pymc-devs/pymc-examples/pull/25))"
```

Comment:
These should be all bullet points in the same cell with a level 2 markdown header. It also looks like the title right before the watermark code cell is missing

Reply:
Done.